utils/spid_blob_storage.ts

```diff
@@ -0,0 +1,92 @@
+import { BlobService } from "azure-storage";
+import * as azureStorage from "azure-storage";
+import { TaskEither, taskify } from "fp-ts/lib/TaskEither";
+
+/**
+ * This method provides all the capabilities to manage an azure append blob for Spid Request/Response auditing.
```

Comment:
```suggestion
 * Provides all the capabilities to manage an azure append blob for Spid Request/Response auditing.
```

("this method" is superfluos)